Casks/v/voiden.rb

```diff
@@ -1,21 +1,22 @@
 cask "voiden" do
+  arch arm: "arm64", intel: "x64"
+
   version "0.10.7"
-  sha256 "87fbabc4122868ed0cc93bd84e18cb2877dab3a2c6442ebe41772d9a17285a57"
+  sha256 :no_check
 
-  url "https://voiden-releases.s3.eu-west-1.amazonaws.com/voiden/darwin/arm64/Voiden-darwin-arm64-#{version}.zip",
+  url "https://voiden-releases.s3.eu-west-1.amazonaws.com/voiden/darwin/#{arch}/Voiden.dmg",
```

Comment:
```suggestion
  url "https://voiden-releases.s3.eu-west-1.amazonaws.com/voiden/darwin/#{arch}/Voiden-darwin-#{arch}-#{version}.zip",
```

We should keep using the versioned URL so we can verify checksums.